#--- a/plugins/tools/panel/panel_settings_overclocking.h.org
#+++ b/plugins/tools/panel/panel_settings_overclocking.h
#@@ -119,7 +119,7 @@
# 
# void set_fdma(char* inifdma, char* oldfdma, char* newfdma)
# {
#-	if(checkbox("UFS912") || checkbox("ATEMIO520") || checkbox("ATEMIO530") || checkbox("SPARK") || checkbox("ATEMIO510") || checkbox("ATEMIO7600") || checkbox("WHITEBOX"))
#+	if(checkbox("UFS912") || checkbox("ATEMIO520") || checkbox("ATEMIO530") || checkbox("SPARK") || checkbox("HS7110") || checkbox("HS8200") || checkbox("WHITEBOX"))
# 		return;
# 
# 	if(isfile("/proc/cpu_frequ/pll1_fdma_bypass") == 0)
#@@ -191,7 +191,7 @@
# 
# 	addscreenrc(panel_overclock, listbox);
# 
#-	if(checkbox("UFS912") || checkbox("ATEMIO520") || checkbox("ATEMIO530") || checkbox("SPARK") || checkbox("ATEMIO510") || checkbox("ATEMIO7600") || checkbox("WHITEBOX"))
#+	if(checkbox("UFS912") || checkbox("ATEMIO520") || checkbox("ATEMIO530") || checkbox("SPARK") || checkbox("HS7110") || checkbox("HS8200") || checkbox("WHITEBOX"))
# 	{
# 		FreqSelvalue = "3841\n12803\n5121\n16643\n17923";
# 		FreqSeltext = "450 (default)\n500\n600\n650\n700";
#@@ -224,7 +224,7 @@
# 		setchoiceboxselection(node, getownconfig(node->name));
# 	}
# 
#-	if(checkbox("UFS912") == 0 && checkbox("ATEMIO520") == 0 && checkbox("ATEMIO530") == 0 && checkbox("SPARK") == 0 && checkbox("ATEMIO510") == 0 && checkbox("ATEMIO7600") == 0 && checkbox("WHITEBOX") == 0)
#+	if(checkbox("UFS912") == 0 && checkbox("ATEMIO520") == 0 && checkbox("ATEMIO530") == 0 && checkbox("SPARK") == 0 && checkbox("HS7110") == 0 && checkbox("HS8200") == 0 && checkbox("WHITEBOX") == 0)
# 	{
# 		node = addlistbox(panel_overclock, listbox, node, 1);
# 		if(node != NULL)
#@@ -266,7 +266,7 @@
# 		}
# 	}
# 
#-	if(checkbox("UFS912") == 0 && checkbox("ATEMIO520") == 0 && checkbox("ATEMIO530") == 0 && checkbox("SPARK") == 0 && checkbox("ATEMIO510") == 0 && checkbox("ATEMIO7600") == 0 && checkbox("WHITEBOX") == 0)
#+	if(checkbox("UFS912") == 0 && checkbox("ATEMIO520") == 0 && checkbox("ATEMIO530") == 0 && checkbox("SPARK") == 0 && checkbox("HS7110") == 0 && checkbox("HS8200") == 0 && checkbox("WHITEBOX") == 0)
# 	{
# 		node = addlistbox(panel_overclock, listbox, node, 1);
# 		if(node != NULL)
--- a/plugins/player/tithek/tithek.c.org
+++ b/plugins/player/tithek/tithek.c
@@ -102,7 +102,7 @@
 	if(checkrealbox("HS7110") == 0 && checkrealbox("HS8200") == 0 && checkbox("UFS912") == 0 && checkrealbox("HS9510") == 0 && checkrealbox("FS9000") == 0 && checkbox("WHITEBOX") == 0 && checkbox("ATEMIO520") == 0 && checkbox("ATEMIO530") == 0)
 	{
 		if(file_exist("/var/bin/audio.elf") || file_exist("/var/swap/bin/audio.elf") || file_exist("/mnt/swapextension/bin/audio.elf"))
-			textbox(_("Message"), _("Alternativ Audio Firmware not working korrekt with all videos (DTSDOWNMIX)!"), _("OK"), getrcconfigint("rcok", NULL), _("EXIT"), getrcconfigint("rcexit", NULL), NULL, 0, NULL, 0, 1000, 200, 0, 0);
+			textbox(_("Message"), _("Alternative Audio Firmware not working korrekt with all videos (DTSDOWNMIX)!"), _("OK"), getrcconfigint("rcok", NULL), _("EXIT"), getrcconfigint("rcexit", NULL), NULL, 0, NULL, 0, 1000, 200, 0, 0);
 	}
 
 	int aktplayerbuffersize = getconfigint("playerbuffersize", NULL);
--- a/titan/security.h.org
+++ b/titan/security.h
@@ -232,9 +232,9 @@
 
 	fread(buf, 1080, 1, fd);
 
-	if(checkbox("ATEMIO510") == 1)
-	{
-		if((buf[1072] & 0xff) == 0x25 && (buf[1073] & 0xff) == 0x29 && (buf[1074] & 0xff) == 0x02 && (buf[1075] & 0xff) == 0xA0)
+	if (checkbox("HS7110") == 1)
+	{
+		if((buf[1072] & 0xff) == 0x25 && (buf[1074] & 0xff) == 0x02 && (buf[1075] & 0xff) == 0xA0)
 		{
 			debug(10, "ResellerId: found (%s) reseller !", getboxtype());
 			free(buf);
@@ -262,9 +262,9 @@
 			return 0;
 		}
 	}
-	else if(checkbox("ATEMIO7600") == 1)
-	{
-		if((buf[1072] & 0xff) == 0x23 && (buf[1073] & 0xff) == 0x03 && (buf[1074] & 0xff) == 0x00 && (buf[1075] & 0xff) == 0xA0)
+	else if (checkbox("HS8200") == 1)
+	{
+		if((buf[1072] & 0xff) == 0x23 && (buf[1074] & 0xff) == 0x00 && (buf[1075] & 0xff) == 0xA0)
 		{
 			debug(10, "ResellerId: found (%s) reseller !", getboxtype());
 			free(buf);
@@ -312,9 +312,9 @@
 			return 0;
 		}
 	}
-	else if(checkbox("ATEVIO7000") == 1)
-	{
-		if((buf[240] & 0xff) == 0x20 && (buf[241] & 0xff) == 0x9 && (buf[242] & 0xff) == 0x00 && (buf[243] & 0xff) == 0x00)
+	else if (checkbox("FS9000") == 1)
+	{
+		if((buf[240] & 0xff) == 0x20 && (buf[242] & 0xff) == 0x00 && (buf[243] & 0xff) == 0x00)
 		{
 			debug(10, "ResellerId: found (%s) reseller !", getboxtype());
 			free(buf);
@@ -322,9 +322,9 @@
 			return 0;
 		}
 	}
-	else if(checkbox("ATEVIO700") == 1)
-	{
-		if((buf[240] & 0xff) == 0x20 && (buf[241] & 0xff) == 0x9 && (buf[242] & 0xff) == 0x03 && (buf[243] & 0xff) == 0x00)
+	else if (checkbox("HS9510") == 1)
+	{
+		if((buf[240] & 0xff) == 0x20 && (buf[242] & 0xff) == 0x03 && (buf[243] & 0xff) == 0x00)
 		{
 			debug(10, "ResellerId: found (%s) reseller !", getboxtype());
 			free(buf);
#--- a/titan/httpdfunc.h.org
#+++ b/titan/httpdfunc.h
#@@ -3916,7 +3916,7 @@
# 	sendstr = ostrcat(sendstr, ", secondinfobar.value", 1 , 0);
# 
# ///////////////////
#-	if(checkbox("ATEMIO7600") != 1)
#+	if (checkbox("HS8200") != 1)
# 	{
# 		ostrcatbig(&buf, "<tr><td><font class=\"label\">", &maxlen, &pos);
# 		ostrcatbig(&buf, _("Don't clear TV on zap (only with fastzap)"), &maxlen, &pos);
#@@ -4895,7 +4895,7 @@
# 	sendstr = ostrcat(sendstr, ", choiceminitv.value", 1 , 0);
# 
# ///////////////////
#-	if(checkbox("ATEMIO510") == 1 || checkbox("UFS912") == 1 || checkbox("ATEMIO7600") == 1 || checkbox("ATEMIO520") == 1 || checkbox("ATEMIO530") == 1 || checkbox("SPARK") == 1 || checkbox("SPARK7162") == 1)
#+	if(checkbox("HS7110") == 1 || checkbox("UFS912") == 1 || checkbox("HS8200") == 1 || checkbox("ATEMIO520") == 1 || checkbox("ATEMIO530") == 1 || checkbox("SPARK") == 1 || checkbox("SPARK7162") == 1)
# 	{
# 		ostrcatbig(&buf, "<tr><td><font class=\"label\">", &maxlen, &pos);
# 		ostrcatbig(&buf, _("Activate CEC"), &maxlen, &pos);
#@@ -4920,7 +4920,7 @@
# 	}
# 
# ///////////////////
#-	if(checkbox("ATEMIO7600") == 1)
#+	if(checkbox("HS8200") == 1)
# 	{
# 		ostrcatbig(&buf, "<tr><td><font class=\"label\">", &maxlen, &pos);
# 		ostrcatbig(&buf, _("SATA Connector"), &maxlen, &pos);
#--- a/titan/titan.c.org
#+++ b/titan/titan.c
#@@ -1196,7 +1196,7 @@
# 	//check to remove preinstalled tpk packages
# 	ret = tpkupdatepre();
# 	addtimer(&checkdate, START, 2000, -1, NULL, NULL, NULL);
#-	if(checkbox("ATEMIO510") == 0)
#+	if(checkbox("HS7110") == 0)
# 		addtimer(&updatevfd, START, 1000, -1, NULL, NULL, NULL);
# 
# //			setvideomode("720", 0);
#
